tree_unique_args/src/ir.rs

```diff
@@ -205,6 +205,17 @@ impl Constructor {
         format!("({without_parens})")
     }
 
+    pub(crate) fn construct_only_fields<F>(&self, f: F) -> String
```

Comment:
Do we need this function? It seems this can be implemented in terms of `filter_map_fields`.

Reply:
The function was a bit more complex at beginning. I can remove it.

Reply:
Done, also collapse the code in rule generation.